src/pages/images-page/images-page.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { Page } from 'pages/page';
+
+export const ImagesPage = () => {
+  return <Page>ImagesPage</Page>;
```

Comment:
once u create routing using `useRouter`, then use `Page` component there, because looks like this is common wrapper for all pages